.idea/LabA2020_2.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should not put any "environment" files to the repository, because, IDEA files, binary jars, and so on are your local files and they may be incompatible with other dev environments (other versions of IDEA, other configuration, other OS).